packages/esm-patient-medications-app/src/api/api.ts

```diff
@@ -1,29 +1,31 @@
 import { useCallback, useMemo } from 'react';
 import useSWR, { mutate } from 'swr';
+import useSWRImmutable from 'swr/immutable';
+import { openmrsFetch, restBaseUrl, useConfig } from '@openmrs/esm-framework';
+import { type FetchResponse } from '@openmrs/esm-framework';
 import { type ConfigObject } from '../config-schema';
-import { type FetchResponse, openmrsFetch, restBaseUrl, useConfig } from '@openmrs/esm-framework';
 import { type OrderPost, type PatientOrderFetchResponse } from '@openmrs/esm-patient-common-lib';
 import { type DrugOrderBasketItem } from '../types';
-import useSWRImmutable from 'swr/immutable';
 
 export const careSettingUuid = '6f0c9a92-6f24-11e3-af88-005056821db0';
 
+const customRepresentation =
+  'custom:(uuid,dosingType,orderNumber,accessionNumber,' +
+  'patient:ref,action,careSetting:ref,previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,' +
+  'orderType:ref,encounter:ref,orderer:(uuid,display,person:(display)),orderReason,orderReasonNonCoded,orderType,urgency,instructions,' +
+  'commentToFulfiller,drug:(uuid,display,strength,dosageForm:(display,uuid),concept),dose,doseUnits:ref,' +
+  'frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,numRefills,dosingInstructions,' +
+  'duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)';
+
 /**
```

Comment:
Additional note: We can abstract the sort logic in the `usePatientOrders` hook into a generic function so that all the hooks can leverage it. Something like:

```tsx
function sortOrdersByDateActivated(orders: Order[]) {
  return orders?.sort(
    (order1, order2) => new Date(order2.dateActivated).getTime() - new Date(order1.dateActivated).getTime(),
  );
}
```

And then in `usePatientOrders`, you could do:

```tsx
const drugOrders = useMemo(() => sortOrdersByDateActivated(data?.data?.results) ?? null, [data]);
```

Then you'd do the same thing for the other hooks.